src/index.ts

```diff
@@ -202,7 +202,11 @@ class ClientApp {
             if ('gcHostOriginQueryParam' in cfg || 'gcTargetEnvQueryParam' in cfg) {
                 this.assertNonEmptyString(cfg.gcHostOriginQueryParam, 'host origin query param name');
                 this.assertNonEmptyString(cfg.gcTargetEnvQueryParam, 'target env query param name');
-                this._pcEnv = this.lookupGcEnv(parsedQueryString[cfg.gcHostOriginQueryParam], parsedQueryString[cfg.gcTargetEnvQueryParam]);
+                const parsedGcHostOrigin = parsedQueryString[cfg.gcHostOriginQueryParam];
+                const parsedGcTargetEnv = parsedQueryString[cfg.gcTargetEnvQueryParam];
+                this.assertNonEmptyString(parsedGcHostOrigin, 'host origin parsed query param');
+                this.assertNonEmptyString(parsedGcTargetEnv, 'target env parsed query param');
```

Comment:
Trying to get the assert to fail but not having any luck with: https://localhost:8443/externalContactsDemo/index.html?gcHostOrigin=https://apps.inindca.com&gcTargetEnv=dev&gcTargetEnv=test

Reply:
I'm able to get it to fail with this, since it's getting a null gcHostOrigin
https://localhost:8443/toast.html?pcEnvironment=inindca.com&pcLangTag=en-us&pcUsePopupAuth=true&gcHostOrigin&gcTargetEnv=dev&gcLangTag=en-us&gcUsePopupAuth=true

Reply:
@carolinepar ... I did confirm that the assertion is thrown and handled by the demo app.  I might have just missed the catch and visualzed error in the status bar.  Been awhile since I tested this demo.

![image](https://github.com/MyPureCloud/client-app-sdk/assets/11987207/4f884e79-412a-4326-b2f8-5e69cecbd363)

I also tested with gcTargetEnv=prod,prod-usw2 and gcTargetEnv=prod&gcTargetEnv=prod-usw2 and this branch matches the existing behavior. So, we should be good with the few small clean-up fixes!